app/assets/javascripts/test/reducers/skeletontracing_reducer.spec.js

```diff
@@ -234,7 +235,11 @@ test("SkeletonTracing should delete nodes and split the tree", t => {
             ]),
             timestamp: Date.now(),
             branchPoints: [{ nodeId: 1, timestamp: 0 }, { nodeId: 7, timestamp: 0 }],
-            edges: [{ source: 0, target: 1 }, { source: 2, target: 1 }, { source: 1, target: 7 }],
+            edges: EdgeCollection.loadFromEdges([
```

Comment:
Unfortunately, I had to change several tests, since the edge order is not really well defined anymore (`DiffableMap` is unordered). I went through all tests manually and all changes seem to be justified. If you want to double check, I'd recommend to click through the UI instead to provoke a bug :)